sky/clouds/kubernetes.py

```diff
@@ -146,7 +146,7 @@ def get_default_instance_type(
         # exactly the requested resources.
         instance_cpus = float(
             cpus.strip('+')) if cpus is not None else cls._DEFAULT_NUM_VCPUS
-        instance_mem = float(memory.strip('+')) if memory is not None else \
+        instance_mem = float(memory.strip('+x')) if memory is not None else \
```

Comment:
Hmm, is this for controller's `memory: 3x`? Why do we need it now but not before? Does it make sense to assign `3` to `instance_mem`?

nit: use parenthesis, not `\`?

Reply:
Ahh, good catch! We should use it to multiply instance_cpus instead.

This is an issue introduced by the PR #3191 and when the user don't have spot controller and has kubernetes enabled.